app/javascript/components/mentoring/discussion/CloseButton.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { GraphicalIcon } from '../../common'
+
+export const CloseButton = ({ url }: { url: string }): JSX.Element => (
+  <a href={url} className="close-btn" aria-label="Close discussion">
```

Comment:
This is fine. It's friendlier to actually tell the screenreader _where_ the user is going to. `Close discussion and go to X`

Reply:
This got fixed in another branch.